core/filters/price.js

```diff
@@ -1,14 +1,21 @@
 import { currentStoreView } from '@vue-storefront/core/lib/multistore'
 
-const formatValue = (value, locale) => {
-  const price = Math.abs(parseFloat(value))
-  const formatter = new Intl.NumberFormat(locale, { minimumFractionDigits: 2, maximumFractionDigits: 2 })
```

Comment:
I see your changes, looks much better but... why not just changing this line? I mean... I think it's totally enough to move just these two options  `minimumFractionDigits` and `maximumFractionDigits` to the config file and you don't anything else... ?

Reply:
Because as said #3553, they also wanted to be able to change the separators.